src/components/header/iframe-pages/small-device.jsx

```diff
@@ -35,7 +35,6 @@ const SmallDeviceExample = () => (
                 placeholder="Find all the answers..."
                 fullWidth
                 color={SEARCH_COLOR.LIGHT}
```

Comment:
`SEARCH_COLOR.LIGHT` is also not exist should be `SEARCH_COLOR.WHITE` according to this
https://github.com/brainly/style-guide/blob/499e637d5ccc49deab32e75d87293e185ba591fa/src/components/form-elements/Input.jsx#L60-L63

Reply:
yes, thank you very much, I removed this prop because otherwise the search icon is not visible (if white)

Reply:
Oh, I think it's visible pretty visible :smiley: 
![image](https://user-images.githubusercontent.com/5789670/74145651-712a3280-4c10-11ea-80cc-5e4a0ed65dcb.png)

Reply:
What do you think @ElaLukaszewicz?

Reply:
yes, sorry for that, it rebuild that way for me 
![image](https://user-images.githubusercontent.com/39903772/74147348-500f1500-4c03-11ea-80ba-08352d3e7ae8.png) so i thought this prop was to change color of the icon, i didn't notice the icons didn't load yet: fix here https://github.com/brainly/style-guide/pull/1725 🤦‍♀

Reply:
Sure, thanks for handling it @ElaLukaszewicz :v: